js/tests.js

```diff
@@ -52,4 +52,47 @@ describe("get auto-complete context", function() {
         expect(parser.autoCompleteContext('Hi @(CONCAT("@con')).toBeNull();
         expect(parser.autoCompleteContext('Hi @("!" & "@con')).toBeNull();
     });
+
+    it("ignore parenthesis triggering functions completions for variables", function() {
+        expect(parser.autoCompleteContext("Hi @(contact.age")).toBe('contact.age');
+    });
+
+    it('ignore the parenthesis triggering functions completions for functions', function() {
+        expect(parser.autoCompleteContext("Hi @(SUM")).toBe('SUM');
+    });
+
+    it("matches the function without parameters", function() {
+        expect(parser.autoCompleteContext("Hi @(SUM(")).toBe('SUM');
```

Comment:
Ohhh, I get what you say, I see now how it broken for that case since they share the "RAND"
I'll find a fix for this case